http/swagger.yml

```diff
@@ -9640,6 +9640,167 @@ components:
           format: float
         legendOrientationThreshold:
           type: integer
+    GeoViewLayer:
+      type: object
+      oneOf:
+        - $ref: "#/components/schemas/GeoCircleViewLayer"
+        - $ref: "#/components/schemas/GeoHeatMapViewLayer"
+        - $ref: "#/components/schemas/GeoPointMapViewLayer"
+        - $ref: "#/components/schemas/GeoTrackMapViewLayer"
+    GeoViewLayerProperties:
+      type: object
+      required: [type, field, colors]
```

Comment:
where are `field` and `colors` defined?

Reply:
Fixed, haven't noticed this since swagger generators do not complain about this problem.